src/main.c

```diff
@@ -55,7 +55,7 @@ static GOptionEntry entries[] = {{"snapd-socket-path", 0, G_OPTION_FLAG_NONE,
 static void do_startup(GObject *object, gpointer data) {
   sdi_snapd_client_factory_set_custom_path(snapd_socket_path);
 
-  refresh_monitor = sdi_refresh_monitor_new(G_APPLICATION(object));
+  refresh_monitor = sdi_refresh_monitor_new();
```

Comment:
This also doesn't look right to me

Reply:
That's because the application isn't used in sdi_refresh_monitor_new(). It was used some PRs before, but in a refactor it wasn't needed, but accidentally forgot to remove it from the parameters, so TIOBE complains that the parameter isn't used in that function.